src/legacy/status_im/data_store/communities.cljs

```diff
@@ -64,6 +64,7 @@
         :canJoin                     :can-join?
         :requestedToJoinAt           :requested-to-join-at
         :isMember                    :is-member?
+        :outroMessage                :outro-message
         :adminSettings               :admin-settings
```

Comment:
would be nice to move this data_store file to it's non legacy counter part :)

e.g src/status_im/data_store/communities.cljs